espnet2/enh/encoder/stft_encoder.py

```diff
@@ -166,3 +198,12 @@ def streaming_frame(self, audio):
         strides = strides + [self.hop_length]
 
         return audio.as_strided(shape, strides, storage_offset=0).unbind(dim=-1)
+
+
+if __name__ == "__main__":
+    enc = STFTEncoder(spec_transform_type="exponent")
+
+    audio = torch.randn((1, 16000))
```

Comment:
Is it OK to embed 16,000 here?

Reply:
If you assume 16000 sampling rate, we need to treat it as a variable and specify it from the option.

Reply:
we can remove this from here as I assume it was used as a quick test from Chenda ;)

Reply:
Yep, it's just a quick test. Removed.